packages/preview/tongji-undergrad-thesis-typst/0.1.0/style/tongjithesis.typ

```diff
@@ -0,0 +1,190 @@
+#import "utils.typ": *
+#import "elements.typ": *
+
+#set pagebreak(weak: true)
+
+#let thesis(
+  school: "某学院",
+  major: "某专业",
+  id: "0000000",
+  student: "某某某",
+  teacher: "某某某",
+  title: "某标题",
+  subtitle: "某副标题",
+  title_english: "Some Title",
+  subtitle_english: "Some Subtitle",
```

Comment:
Typst packages generally use kebab-case names, not snake_case (it is not mandatory however, you can leave it as is if you prefer).